databricks_cli/jobs/cli.py

```diff
@@ -113,39 +135,62 @@ def list_cli(api_client, output):
 
     In table mode, the jobs are sorted by their name.
     """
-    jobs_api = JobsApi(api_client)
-    jobs_json = jobs_api.list_jobs()
-    if OutputClickType.is_json(output):
-        click.echo(pretty_format(jobs_json))
+    api_version = version or api_client.jobs_api_version
+    if api_version != '2.1' and (expand_tasks or offset or limit or _all):
+        click.echo(click.style('ERROR', fg='red') + ': the options --expand-tasks, ' +
+                   '--offset, --limit, and --all are only available in API 2.1', err=True)
     else:
-        click.echo(tabulate(_jobs_to_table(jobs_json), tablefmt='plain', disable_numparse=True))
+        jobs_api = JobsApi(api_client)
+        has_more = True
+        count = 0
+        jobs = []
+        jobs_json = {}
+        while has_more:
+            if count > 0:
+                offset = (offset or 0) + \
```

Comment:
This might be cleaner if we just increase the offset after the API call? This arithmetic is hard to read.